From 65b298c7ca317d7e4316aa2b9e0499e13047c65c Mon Sep 17 00:00:00 2001
From: Alan Coopersmith <alan.coopersmith@oracle.com>
Date: Sat, 6 Feb 2016 12:26:21 -0800
Subject: [PATCH 3/3] Correct @param "e" to "error" in xcb_poll_for_reply*()

Found by clang -Wdocumentation:

./xcbext.h:271:11: warning: parameter 'e' not found in the function
      declaration [-Wdocumentation]
 * @param e Location to store errors in, or NULL. Ignored for un...
          ^
./xcbext.h:271:11: note: did you mean 'error'?
 * @param e Location to store errors in, or NULL. Ignored for un...
          ^
          error

./xcbext.h:283:11: warning: parameter 'e' not found in the function
      declaration [-Wdocumentation]
 * @param e Location to store errors in, or NULL. Ignored for un...
          ^
./xcbext.h:283:11: note: did you mean 'error'?
 * @param e Location to store errors in, or NULL. Ignored for un...
          ^
          error

Signed-off-by: Alan Coopersmith <alan.coopersmith@oracle.com>
Signed-off-by: Uli Schlachter <psychon@znc.in>
---
 src/xcbext.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/xcbext.h b/src/xcbext.h
index bb422f0..90f9d58 100644
--- a/src/xcbext.h
+++ b/src/xcbext.h
@@ -268,7 +268,7 @@ void *xcb_wait_for_reply64(xcb_connection_t *c, uint64_t request, xcb_generic_er
  * @param c The connection to the X server.
  * @param request Sequence number of the request as returned by xcb_send_request().
  * @param reply Location to store the reply in, must not be NULL.
- * @param e Location to store errors in, or NULL. Ignored for unchecked requests.
+ * @param error Location to store errors in, or NULL. Ignored for unchecked requests.
  * @return 1 when the reply to the request was returned, else 0.
  *
  * Checks if the reply to the given request already received. Does not block.
@@ -280,7 +280,7 @@ int xcb_poll_for_reply(xcb_connection_t *c, unsigned int request, void **reply,
  * @param c The connection to the X server.
  * @param request 64-bit sequence number of the request as returned by xcb_send_request().
  * @param reply Location to store the reply in, must not be NULL.
- * @param e Location to store errors in, or NULL. Ignored for unchecked requests.
+ * @param error Location to store errors in, or NULL. Ignored for unchecked requests.
  * @return 1 when the reply to the request was returned, else 0.
  *
  * Checks if the reply to the given request already received. Does not block.
-- 
2.8.3

